### DISCUSSION TITLE
KeySmith & Apple Shortcut Support

### DISCUSSION THREAD

**Discussion by Jonlb21** [Anze/KeyCluCask#85] [Ideas]
I would love the ability to get support for Keysmith at some point. Keysmith lets you easily create shortcuts and macros for MacOS https://www.keysmith.app/. It is drastically easier to use to compared to something like Keyboard Maestro and just as powerful. I use Keysmith to create dozens and dozens or macro actions to be used on my Loupedeck. I probably have close to 100 created throughout different apps. It would be great if those could show in KeyCu when selecting the corresponding app. I would greatly appreciate it. 

KeyCu might already support this, and I'm not sure how to view it. But I wanted to request an option for Apple Shortcuts . When I trigger Keycu in the shortcuts app, none of my hotkeys are displayed for my different shortcuts. It would be super helpful to have that added at some point.

**Comment by Anze** [comment]
Hey [Jonlb21](https://github.com/Jonlb21),

Thanks again for the interest.

I’ll check Keysmith later to see if it can be included in the supported integrations.
As for Apple Shortcuts, I don’t have much experience with it, but I’ll look into it as well.

**Comment by Anze** [comment]
Hey [Jonlb21](https://github.com/Jonlb21),
I took a quick look at the Keysmith, and looks like it stores its data in database files that are password-protected. It's unusual compared to other apps where the data is more accessible. I've contacted the developers, and they confirmed that there are no plans to make the file accessible to users. Unfortunately, because of this protection, it's unlikely that integration would be possible.
